service/services/eventDiscoveryService/src/test/java/com/event/discovery/agent/integrationTests/offline/kafkaCommon/kafka/KafkaEventDiscoveryTestsIT.java

```diff
@@ -371,6 +372,27 @@ public void testAclError() throws Exception {
         assertThat(status.getStatus(), is(JobStatus.ERROR));
     }
 
+    @Test
```

Comment:
Following TDD principals, this is the test that was first added, which failed, but then passed once the fix was added to the application.